src/set_theory/ordinal_arithmetic.lean

```diff
@@ -409,6 +412,9 @@ theorem is_normal.is_limit {f} (H : is_normal f) {o} (l : is_limit o) :
 λ a h, let ⟨b, h₁, h₂⟩ := (H.limit_lt l).1 h in
   lt_of_le_of_lt (succ_le.2 h₂) (H.lt_iff.2 h₁)⟩
 
+theorem is_normal.self_le_iff_eq {f} (H : is_normal f) {a} : f a ≤ a ↔ f a = a :=
```

Comment:
The `self` looks misplaced.

Reply:
What might be a better name? `is_normal.le_iff_eq`?

Reply:
Yeah, or `le_self_iff_eq` if you need to disambiguate.